pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[build-system]
+requires = ["setuptools >= 64"]
+build-backend = "setuptools.build_meta"
```

Comment:
Can you elaborate on why this is necessary?

Reply:
pyproject.toml is the defacto standard now and all projects are supposed to have one to select which build system is used - without it editable installs complain and will soon be disallowed (because the editable part needs to know which build system to use to install editable).

Reply:
unfortunately in order to support stuff like api mode it's required, migration from .cfg to pyproject.toml is pretty doable, but feels like it might want to be separate to this PR.